Makefile

```diff
@@ -37,22 +37,22 @@ compress:
 	upx --brute -1 ./dist/fuseml-windows-amd64
 	upx --brute -1 ./dist/fuseml-darwin-amd64
 
-test:
+test: embed_files
 	ginkgo ./cmd/internal/client/ ./tools/ ./helpers/ ./kubernetes/
 
-test-acceptance-traefik:
+test-acceptance-traefik: embed_files
 	@./scripts/test-acceptance.sh -- -serve=deployment
 
-test-acceptance-knative:
+test-acceptance-knative: embed_files
 	@./scripts/test-acceptance.sh -- -serve=knative
 
-test-acceptance-kfserving:
+test-acceptance-kfserving: embed_files
 	@./scripts/test-acceptance.sh -- -serve=kfserving
```

Comment:
Do we need to call embed files here? the test calls `build`, which already includes embed_files.

Reply:
Where they call `build`?

Reply:
see: https://github.com/fuseml/fuseml/blob/main/acceptance/acceptance_suite_test.go#L211

Reply:
But that's probably too late because some dependency required statik earlier. Let me show you by removing it

Reply:
Yep. The file imports `helpers` which means also `helpers/embedded_files.go` which requires built `statik` content...
https://github.com/fuseml/fuseml/pull/34/checks?check_run_id=2138381748

Reply:
ok, thanks for checking 😃